src/struct/ClientUtil.js

```diff
@@ -398,6 +398,19 @@ class ClientUtil {
     collection(iterable) {
         return new Collection(iterable);
     }
+
+    /**
+     * Makes a WebhookClient.
+     * @param {string} [link] - The webhook link.
+     * @returns {Collection}
+     */
+    webhook(link) {
+        const parts = link.split('/');
+        const id = parts[parts.length - 2];
+        const token = parts[parts.length - 1];
+        if (!id || !token) return null;
+        return new WebhookClient(parts[parts.length - 2], parts[parts.length - 1]);
```

Comment:
```suggestion
        return new WebhookClient(id, token);
```
You already have token and id stored in variables, so there's no need to pull from the array again